cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -284,6 +311,51 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
         }
     }
 
+    /**
+     * Determine if a package should be included in the download. A package may not be downloaded it the license of the
+     * package is categorized as a category that is not included in the [DownloaderConfiguration]'s
+     * `includedLicenseCategories`
+     */
+    // TODO: Find a way how to get rid of the `ortResult` parameter which is currently needed for the license choices.
+    private fun isLicenseCategoryIncluded(
+        pkg: Package,
+        ortResult: OrtResult,
+        licenseCategorizations: List<LicenseCategorization>
+    ): Boolean {
+        val includedLicenseCategories = globalOptionsForSubcommands.config.downloader.includedLicenseCategories
+
+        if (includedLicenseCategories.isEmpty()) return true
```

Comment:
We could actually make this check also part of line 221 already. As then not much is left in this function, I wonder whether we should inline the remainders into the `filter` in line 225.